.jscsrc

```diff
@@ -0,0 +1,13 @@
+{
+  "validateIndentation":2,
+  "disallowTrailingWhitespace": true,
+  "disallowTrailingComma": true,
+  "disallowNewlineBeforeBlockStatements": true,
+  "requireSpaceAfterBinaryOperators": true,
+  "requireCommaBeforeLineBreak": true,
+  "requireSpaceBeforeBinaryOperators": true,
+  "requireSpaceAfterBinaryOperators": true,
+  "disallowSpaceAfterPrefixUnaryOperators": true,
+  "disallowSpacesInsideObjectBrackets": true,
+  "disallowSpacesInsideArrayBrackets": true
+}
```

Comment:
@scottnath Can we delete this file until we agree on what set of coding standards we'll be using going forward?